site/plugins.md

```diff
@@ -390,8 +391,257 @@ func (p *Plugin) Report(w http.ResponseWriter, r *http.Request) {
 
 ```
 
-**See Also**
+### <a id="report-data-structures"></a>Report Data structures
+A report can contain many types of information.
+If you go back to the [Reporter Interface](#reporter-interface) section, you will see the top-level `Plugins` attribute.
+Along with that, a report may contain multiple topologies.
+An example of a report containing a few topologies is the following:
 
-  * [Building Scope](/site/building.md)
+```json
+{
+  "Host": {},
+  "Container": {},
+  "Process": {},
+  ...,
+  "Plugins": [...,]
+}
+```
+
+### Topologies
+Topologies can be of various types.
+Each topology consists of some general information and a list of nodes.
+These are the available topologies:
+
```

Comment:
I don't think you need "Topologies can be of various types" 

A topology consists of general information along with a list of nodes. 

(What is the general information specifically meta-data?)